debian/raspberrypi-sys-mods.run_once.service

```diff
@@ -0,0 +1,18 @@
+# This unit is heavily inspired by Ned McClain's ideas found in https://github.com/nmcclain/raspberian-firstboot
+[Unit]
+Description=run_once initialization
+After=network.target
```

Comment:
per https://github.com/nmcclain/raspberian-firstboot/issues/13 ...

Update `After=network.target` to include `apt-daily.service` and `apt-daily-upgrade.service`

Reply:
Seems reasonable

How does this behave on a system without network connectivity?

Reply:
`network.target` is a passive unit, hence it has no meaning unless a network stack/provider gives it one (sorts itself `Before=network.target`). In case of Raspbian/Debian (Raspberry Pi OS), this is most of all the `networking.service` from the `ifupdown` package, which brings up the loopback interface and other manually configured `auto <iface>` interfaces. Theoretically it's as well the `ifup@<iface>.service`'s, which are created for `allow-hotplug <iface>` interfaces, but since this unit is of `Type=simple`, they don't need to finish but only need to start (hence do not delay the target to be reached, even if e.g. DHCP or wpa_supplicant hangs or fails).

`dhcpcd` does NOT have any effect on this target, which is strange and somehow a problem since Raspberry Pi OS uses it for network configuration by default 🤔.

So if no network is configured, the target is reached once the loopback interface is available. If a network is configured, it depends on the network stack, but the way Raspberry Pi OS is configured, no additional delay happens.

Btw, `network.target` is not intended to indicate that the device is online. For this, `network-online.target` is intended. Since it is an active target, it needs to be pulled it:
```
Wants=network-online.target
After=network-online.target
```
But currently (with the current network stacks) it has not the intended meaning but simply is reached immediately after `network.target`.

More info: https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
- But it has to be taken care since the is quite some discrepancy between the intended meaning by systemd and the actual meaning on the system, which depends on network stack and consumer services.
- The effective meaning can be best seen with:
    ```sh
    systemctl show -p Before,After network.target network-online.target
    ```